### DISCUSSION TITLE
End subscription MaxMsgs

### DISCUSSION THREAD

**Discussion by alberk8** [nats-io/nats.net#470] [Q&A] [answered]
I have the code below and after the request is returned I get the above message logged.  What does this mean?

` var ret = await _natsConnection.RequestAsync<int, List<Person>>("Person.GetPerson", 9912 );`

`[09:48:17.148  DBG][0HN2QI688TOR4:00000006] NATS.Client.Core.NatsSubBase: End subscription MaxMsgs`

**Comment by mtmk** [comment, marked as answer]
That's to be expected. When using request response we use a subscription with max messages set to 1 since we only expect a single response.
